peeringdb_server/search.py

```diff
@@ -107,39 +147,34 @@ def search(term, autocomplete=False):
         search_query = make_autocomplete_query(term).models(*autocomplete_models)
         limit = settings.SEARCH_RESULTS_AUTOCOMPLETE_LIMIT
     else:
-        search_query = make_search_query(term).models(*searchable_models)
```

Comment:
Wondering why the `models` call was removed - if i recall correctly there were some performance gains in the search query time when limiting which models are included in the search. 

It seems like the `searchable_models` list would still cover all the models needed for the new improvements?

thanks!